interface/resources/qml/hifi/avatarapp/AvatarsModel.qml

```diff
@@ -21,17 +21,47 @@ ListModel {
 
         return marketItemUrl;
     }
+	
+    function makeMarketThumbnailUrl(marketId) {
+        var avatarThumbnailUrl = "https://hifi-metaverse.s3-us-west-1.amazonaws.com/marketplace/previews/%marketId%/large/hifi-mp-%marketId%.jpg"
+            .split('%marketId%').join(marketId);
+            
+        return avatarThumbnailUrl;
+    }
+	
+    function trimFileExtension(url) {
+        var trimmedUrl = url.substring(0, (url.indexOf("#") == -1) ? url.length : url.indexOf("#"));
+        trimmedUrl = trimmedUrl.substring(0, (trimmedUrl.indexOf("?") == -1) ? trimmedUrl.length : trimmedUrl.indexOf("?"));
+        trimmedUrl = trimmedUrl.substring(0, trimmedUrl.lastIndexOf("."));
+
+        return trimmedUrl;
+    }
+	
+    function imageExists(image_url){
```

Comment:
image_url has inconsistent naming,  please use camelCase  e.g. `imageUrl`   ,   also space is missing between ) and {